### PR TITLE
Scala 3 support

### PR DESCRIPTION
**Pull request by grzegorz-bielski**
Forum thread: https://forum.exercism.org/t/scala-3-support/12117

Blocked by: https://github.com/exercism/scala/pull/836

### CODE REVIEW

**Review by ErikSchierboom — COMMENTED** [review]
This is looking good!

File: src/main/scala/Application.scala [hunk 1]
```diff
@@ -1,80 +1,93 @@
...
-  def toExercismJSON(buildLogFilePath: String, testResultsFiles: List[File]): JSONObject = {
-    val baseObject = new JSONObject().put("version", 2)
+  def toExercismJSON(
+    buildLogFilePath: String,
+    testResultsFiles: List[File],
+  ): JSONObject =
+    val baseObject   = new JSONObject().put("version", 2)
     val errorMessage = findErrorsInLog(buildLogFilePath)
-    if (errorMessage.nonEmpty) {
+    println("errorMessage" -> errorMessage)
```
Comment (ErikSchierboom) [inline src/main/scala/Application.scala:69]: Is this line still needed?

Comment (grzegorz-bielski) [reply src/main/scala/Application.scala]: Nope, removed. Thx!

**Comment by ErikSchierboom** [comment]
Great work! We'll need to synchronize the merging of this PR with the merging of the other PR.

**Comment by ErikSchierboom** [comment]
@grzegorz-bielski When would be a good time for you to merge both PRs?

**Comment by grzegorz-bielski** [comment]
@ErikSchierboom For me they could be merged at any moment, but I believe I lack proper permissions to do so.

**Pull request merged**